ansible/roles/ml-data-pipeline/tasks/main.yml

```diff
@@ -0,0 +1,115 @@
+---
+# tasks file for data-pipeline
+- name: Git checkout
+  git:
+    repo: 'https://github.com/Sunbird-Ed/ml-analytics-service.git'
+    dest: "/opt/sparkjobs/ml-analytics-service"
+    version: "{{ ml_analytics_version }}"
+
+- name: CREATE THE USER
+  user:
+    name: "{{ USER }}"
+    state: present
+    home: "/home/{{ USER }}"
+    shell: /bin/bash
+
+- name: INSTALL THE PACKGES
+  apt:  
+    update_cache: yes
+    name: "{{ item }}"
+    state: present
+  loop:
+    - openjdk-8-jdk
+    - software-properties-common
+    - python3-pip
+    - python3.6-dev
+    - python3.6-venv
+    - python-virtualenv
+    - zip
+    - unzip
+
+- name: CHANGE THE OWNERSHIP FOR THIS {{ BASEPATH }} DIRECTORY
+  file:
+    path: "{{ item }}"
+    mode: "0755"
+    recurse: yes
+    state: directory
+    owner: "{{ USER }}"
+    group: "{{ USER }}"
+  loop:
+    - "{{ BASEPATH }}"
+    - "{{ WORKDIR }}"
+    - "{{ WORKDIR }}/faust_as_service"
```

Comment:
If we doing a recurisive ownership change, why is this line required? This will already get covered from the previous line right?

Reply:
faust_as_service directory has been created here only so we can not able to remove this part